docs/incidents/2019-02-06-multiple-file-uploads.md

```diff
@@ -0,0 +1,79 @@
+# 2019-02-06
+
+**Incident Leader: Peter**
+
+## Description
+
+Something went wrong and caused this problem.
+
+## Timeline
+
+All times UTC.
+
+10:00 standup in which the overwriting file name issues is discussed and parked in favor of parallel file uploads
```

Comment:
@hdrury1 ... my recollection is that the whole thing was parked in favour of the playbook.

Reply:
Also there was only one issue at this time.

Reply:
My take: we agreed to park overwriting filenames in the knowledge that this would be more easily achieved after refactoring. There was a question around whether or not the files were supposed to upload sequentially and it was confirmed that this had always been the intention.